api/Element.json

```diff
@@ -4391,6 +4495,422 @@
           }
         }
       },
+      "mousedown_event" {
```

Comment:
```suggestion
      "mousedown_event": {
```

Reply:
I already made the change and pushed it, but thanks!